tests/SideBySide/ConnectAsync.cs

```diff
@@ -178,6 +179,102 @@ public ConnectAsync(DatabaseFixture database)
 			}
 		}
 
+		[Fact]
```

Comment:
Especially if one were to add a [config.json](https://github.com/mysql-net/MySqlConnector/pull/204/files#diff-35465726ecae7ec93dfc2a4abc27dd03) that didn't have this setting... :smile: